src/ukv/ukv.hpp

```diff
@@ -0,0 +1,411 @@
+#pragma once
+
+#include <cassert>
+
+#include <ukv/cpp/status.hpp>
+#include <ukv/cpp/types.hpp>
+#include <ukv/ukv.h>
+
+#include "src/core/db.hpp"
+#include "src/core/helper.hpp"
+#include "src/core/types.hpp"
+namespace ucsb::ukv {
+
+namespace fs = ucsb::fs;
+using namespace unum::ukv;
+
+using key_t = ucsb::key_t;
+using keys_spanc_t = ucsb::keys_spanc_t;
+using value_span_t = ucsb::value_span_t;
+using value_spanc_t = ucsb::value_spanc_t;
+using values_span_t = ucsb::values_span_t;
+using values_spanc_t = ucsb::values_spanc_t;
+using value_lengths_spanc_t = ucsb::value_lengths_spanc_t;
+using operation_status_t = ucsb::operation_status_t;
+using operation_result_t = ucsb::operation_result_t;
+using db_hints_t = ucsb::db_hints_t;
+using transaction_t = ucsb::transaction_t;
+
+thread_local arena_t arena(nullptr);
```

Comment:
Isn't this supposed to be a `ukv_arena_t`? Probably better to put it in the `private` scope of the `class`.